apps/api/src/variable/service/variable.service.ts

```diff
@@ -604,6 +604,39 @@ export class VariableService {
     return Array.from(variablesWithEnvironmentalValues.values())
   }
 
+  async getRevisionsOfVariable(
+    user: User,
+    variableId: Variable['id'],
+    environmentId: Environment['id'],
+    page: number,
+    limit: number
+  ) {
+    await this.authorityCheckerService.checkAuthorityOverVariable({
+      userId: user.id,
+      entity: { id: variableId },
+      authority: Authority.READ_VARIABLE,
+      prisma: this.prisma
+    })
+
+    await this.authorityCheckerService.checkAuthorityOverEnvironment({
+      userId: user.id,
+      entity: { id: environmentId },
+      authority: Authority.READ_ENVIRONMENT,
+      prisma: this.prisma
+    })
+
+    const revisions = await this.prisma.variableVersion.findMany({
+      where: {
+        variableId: variableId,
+        environmentId: environmentId
+      },
+      skip: page * limit,
+      take: limit
```

Comment:
Could you also include the `order` field in here? Ideally, we would like to sort based on `order` on the `version` field.